CHANGELOG.md

```diff
@@ -6,6 +6,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## unreleased
 
+### Removed
+
+- Remove `derive(DialogueState)` macro
+
+### Changed
+
+- `#[command(rename = "a_name_that_is_not_a_case_name")]` doesn't work anymore
+- A lot of internal refactoring
```

Comment:
Not sure that it needs to be in the changelog since "refactoring" doesn't reflect any API changes.